flent/tests/http-1up.conf

```diff
@@ -16,6 +16,19 @@ PLOTS['totals'] = {'description': 'TCP bandwidth with delay and jitter',
                    'type': 'meta',
                    'subplots': ('bandwidth', 'http_delay')}
 PLOTS['http_delay_cdf']['cutoff'] = (DELAY,-DELAY)
+
+PLOTS['box_total'] = {'description': 'Candlestick plot of HTTP latency and TCP bandwidth',
```

Comment:
We don't call them "candlestick plots" anywhere else, so let's stick with "box plot" in the description to be consistent...